src/storm/automata/LTL2DeterministicAutomaton.cpp

```diff
@@ -20,7 +20,7 @@
 namespace storm {
 namespace automata {
 
-std::shared_ptr<DeterministicAutomaton> LTL2DeterministicAutomaton::ltl2daSpot(storm::logic::Formula const& f, bool dnf) {
+std::shared_ptr<DeterministicAutomaton> LTL2DeterministicAutomaton::ltl2daSpot([[maybe_unused]] storm::logic::Formula const& f, [[maybe_unused]] bool dnf) {
```

Comment:
Do we want ot move the ifdef outside of the definition to prevent potential uncaught problems due to the maybe unused? I am also fine if this is the way we do this, just asking!